src/btree/bt_curnext.c

```diff
@@ -334,8 +334,8 @@ __cursor_var_next(WT_CURSOR_BTREE *cbt, bool newpage, bool restart, size_t *skip
  *     Move to the next row-store item.
  */
 static inline int
-__cursor_row_next(
-  WT_CURSOR_BTREE *cbt, bool newpage, bool restart, size_t *skippedp, WT_ITEM *prefix)
+__cursor_row_next(WT_CURSOR_BTREE *cbt, bool newpage, bool restart, bool *prefix_key_out_of_bounds,
```

Comment:
Should we have `bool *prefix_key_out_of_bounds` as the last arg after `WT_ITEM *prefix`?

Reply:
I thought the prefix_key_out_of_bounds variable was already at the end with all the other pointers, but I can put it as the very last variable if that's the standard?

Reply:
I usually put the outputs at the end of the function args, I think `prefix` which is currently at the end is an input.
@quchenhao, is the current place of `prefix_key_out_of_bounds` correct?

Reply:
I will move it to the last argument, that is what I see in other code samples